### DISCUSSION TITLE
Welcome to react-anime-manager Discussions!

### DISCUSSION THREAD

**Discussion by perymimon** [perymimon/React-Anime-Manager#4] [Announcements]
## 👋 Welcome!
  I use Discussions as a place to connect with my users of the pack. I hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this code.

**Comment by ordant** [comment]
Sounds Awesome !! I can't wait to try it
